src/qt/guiutil.cpp

```diff
@@ -399,7 +400,13 @@ bool openBitcoinConf()
     configFile.close();
 
     /* Open bitcoin.conf with the associated application */
-    return QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+    bool res = QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+#if defined(Q_OS_MAC)
```

Comment:
nit: `#ifdef` is more common in Bitcoin Core code base ;)

Reply:
fixed in the new commit